.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+# push will run on every pushed commit to any branch (so this will rerun the tests
+# once a branch gets merged to main in addition to any new commits on any branch)
+on: push
+
+name: CI
+
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  Security:
+    name: Security Pipeline
+    uses: uc-cdis/.github/.github/workflows/securitypipeline.yaml@master
+    with:
+       python-poetry: 'true'
+    secrets: inherit
+
+  UnitTest:
+    name: Python Unit Test with Postgres
+    uses: uc-cdis/.github/.github/workflows/python_unit_test.yaml@master
+    with:
+       test-script: 'test/ci_commands_script.sh'
```

Comment:
should we also delete the travis file so that we don't run the tests twice? https://github.com/uc-cdis/cdis-python-utils/blob/master/.travis.yml

and in that case i think we're missing a workflow to deploy to pypi

Reply:
oh shoot, you're right